styles.css

```diff
@@ -0,0 +1,872 @@
+    text-transform: uppercase;
+    font-family: 'Roboto', sans-serif;
+    font-weight: 700;
+    font-size: 30px;
+    letter-spacing: 1px;
+}
+
+.footer__copyright-paragraph {
+    text-align: center;
+    color: var(--main-font-color-light);
+    font-size: 10px;
+    font-weight: 300;
+    letter-spacing: 1px;
+}
+
+.services__block--paragraph {
+    color: var(--main-font-color-medium);
+    margin-top: 5em;
+    font-size: 13px;
+}
+
+.footer__form--header {
+    color: var(--main-font-color-light);
+    font-size: 4.5em;
+    margin: 0.5em;
+    text-align: center;
+}
+
+.footer__form--paragraph {
+    text-align: center;
+    color: var(--main-font-color-light);
+    font-weight: 300;
+    font-size: 40px;
+    margin: 1em 0;
+}
+
+.footer__form--form-section {
+    color: var(--main-font-color-light);
+    font-size: 30px;
+}
+
+.form__input[type=text],
+.form__input--text-area {
+    font-family: Roboto, sans-serif;
+    font-size: 30px;
+}
+
+.footer__copyright-paragraph {
+    font-size: 25px;
+}
+
+/* ------------------------------- 3. Layout --------------------------------------- */
+
+.main__inner-container a:link{
+    color: var(--main-focus-color);
+}
+
+.main__inner-container a:visited{
+    color: var(--main-shadow-focus-color);
+}
+
+.main__outer-container {
+    display: flex;
+    /*let op hier row in column verandert voor de triangles!!!*/
+    flex-direction: column;
+    justify-content: center;
+    /*align-items: stretch;*/
+    width: 100%;
+    /*let op hier position relative toegevoegd voor triangles!!!*/
+    position: relative;
+}
+
+.main__cta-button,
+.main__cta-button--footer {
+    background-color: var(--main-button-color);
+    padding: 40px 90px;
+    margin: 20px;
+    border: none;
+    border-radius: 60px;
+    color: var(--main-font-color-light);
+    text-transform: uppercase;
+    cursor: pointer;
+    /*font-size: 12px;*/
+    font-size: 29px;
+    box-shadow: 0 8px 10px var(--main-shadow-focus-color);
+    z-index: 1;
+}
+
+.main__cta-button--footer {
+    padding: 45px 140px;
+}
+
+/* ------------------------------- 4. Areas --------------------------------------- */
+
+/* --------------- 4.1 Header ------------------- */
+
+.header__outer-container,
+.projects__header--outer-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: stretch;
+    background-image: var(--main-gradient-color);
+    color: var(--main-font-color-light);
+}
+
+.header__inner-container,
+.projects__header--inner-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 67vh;
+}
+
+
+/* --------------- 4.1b Header-Projects-page ------------------- */
+
+.projects__header--inner-container {
+    justify-content: center;
+    height: 98vh;
+}
+
+/* --------------- 4.2 Header-Navigation ------------------- */
+.navigation {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    justify-content: space-between;
```

Comment:
Als je bij justify content een flex-end had neergezet, dan lijnt de nav zich uit naar links (de plek waar we de navbar wilde hebben)